### PR TITLE
Localtransport

### PR DESCRIPTION
**Pull request by h4ck3rm1k3**
Here is a small hack used during testing, a local transport.
also the raetchannel include is moved to where it is used so the failure is imporant.

### CODE REVIEW

File: salt/transport/__init__.py [hunk 1]
```diff
@@ -149,6 +148,7 @@ def _setup_stack(self, ryn='manor'):
             log.error(emsg + '\n')
             raise ValueError(emsg)
 
+        assert nacling
```
Comment (thatch45) [inline salt/transport/__init__.py]: we generally avoid putting in asserts

**Comment by thatch45** [comment]
Go Go Jenkins!

**Comment by thatch45** [comment]
Looks like there are some lint failures but Jenkins seems to have deleted them, so I am re-running....

File: salt/transport/__init__.py [hunk 2]
```diff
@@ -308,3 +308,36 @@ def send(self, load, tries=3, timeout=60):
...
+    """
+    def __init__(self, opts, **kwargs):
+        self.opts = opts
+        self.kwargs = kwargs
+        self.tries = 0
+
+    def send(self, load):
+
+        if self.tries ==0 :
+            print "load", load
```
Comment (cachedout) [inline salt/transport/__init__.py]: No print statements, please. If we need to see data, please use the appropriate logging functions.

File: salt/transport/__init__.py [hunk 3]
```diff
@@ -308,3 +308,36 @@ def send(self, load, tries=3, timeout=60):
             return self._uncrypted_transfer(load, tries, timeout)
         else:  # for just about everything else
             return self._crypted_transfer(load, tries, timeout)
+
+class LocalChannel(Channel):
+    """
+    Local channel for testing purposes
+    """
```
Comment (cachedout) [inline salt/transport/__init__.py]: No double-quotes, please. See the convention guide for additional details. Thanks! http://docs.saltstack.com/en/latest/topics/development/conventions/style.html

**Comment by h4ck3rm1k3** [comment]
Alright,  thank you!

**Pull request merged**